### PR TITLE
🔀 Merge feat/restrictions to main

### PR DESCRIPTION
**Pull request by cuprum-acid**

### CODE REVIEW

File: src/lib/components/AddTransaction.svelte
Comment (dprostiruk) [inline src/lib/components/AddTransaction.svelte:1]: Got some questions rn😒
If I want to add some transaction before 2014?😢
And what about 1 million rubles?😒

**Review by axonstan — CHANGES_REQUESTED** [review]
Remove date limits

**Review by dprostiruk — APPROVED** [review]
Not bad, but for me it is not reliable😢

File: src/lib/components/AddTransaction.svelte
Comment (dprostiruk) [inline src/lib/components/AddTransaction.svelte:1]: Nice work😁 Limits are reliable😁

**Review by dprostiruk — APPROVED** [review]
Good job😁 In 100 years project will be actual🎉😍

**Comment by cuprum-acid** [comment]
Last commit [9c2dbed](https://github.com/PodYapolskiy/mydget/pull/55/commits/9c2dbed9f79eb4c56809f6596c2cf12e40975587) closes #57

**Review by dprostiruk — APPROVED** [review]
Good job👍 Pretty nice commit😁

File: src/lib/components/AddTransaction.svelte [hunk 1]
```diff
@@ -49,53 +82,68 @@
     // });
 </script>
 
+<dialog id="my_modal_1" class="modal">
```
Comment (axonstan) [inline src/lib/components/AddTransaction.svelte:85]: Id "my_modal_1" does not look reliable😁

**Review by axonstan — APPROVED** [review]

File: src/lib/components/AddTransaction.svelte (hunk 1 above)
Comment (cuprum-acid) [reply src/lib/components/AddTransaction.svelte:85]: This is some [daisyUI](https://daisyui.com/) conventions 😄 
[click](https://daisyui.com/components/modal/)

**Pull request merged**